bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/LocationsIndexRebuildOp.java

```diff
@@ -31,6 +31,9 @@
 import java.util.Date;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicReference;
+
```

Comment:
new checkstyle not pass, please clean line:36 newline

<img width="1382" alt="image" src="https://user-images.githubusercontent.com/42990025/187113881-cb0191bf-3ecd-403b-b23f-e5b94d024e84.png">